src/test/java/org/cactoos/func/RetryTest.java

```diff
@@ -74,4 +73,29 @@ public void runsFuncConditionMultipleTimes() {
             new FuncApplies<>(true, true)
         ).affirm();
     }
+
+    @Test
+    public void processInterruptExceptionOnWait() {
+        final Thread main = Thread.currentThread();
+        new Thread(
+            () -> {
+                try {
+                    Thread.sleep(500);
```

Comment:
Using Thread.sleep in tests is a bad idea. It not only slows down the test suite run time, but it also creates fragile tests that can pass or fail depending on the environment timing.